Source/Turbo Navigator/Extensions/VisitProposalExtension.swift

```diff
@@ -59,4 +59,13 @@ public extension VisitProposal {
 
         return VisitableViewController.pathConfigurationIdentifier
     }
+    
+    /// Allows the proposal to change the animation status when pushing, popping or presenting.
+    var animated: Bool {
+        if let animated = parameters?["animated"] as? Bool {
+            return animated
+        }
+        
+        return true
+    }
```

Comment:
```suggestion
    var animated: Bool { parameters?["animated"] as? Bool ?? true }
```

Reply:
I'd like to keep it as-is since it's consistent across all other functions in the file.